lib/puppet/type/tpm2_ownership.rb

```diff
@@ -0,0 +1,141 @@
+# The tpm_ownership type allows you to take ownership of tpm0.
+#
+# @!puppet.type.param ownerauth TPM owner password. Required.
+# @!puppet.type.param lockauth TPM  lock out password. Required.
+# @!puppet.type.param endorseauth TPM endorsement hierachy password. Required.
+#
+# @!puppet.type.param inhex If true, indicates the passwords are in Hex.
+#
+# @!puppet.type.param local If true, the provider will drop the owner
```

Comment:
I think the name of this parameter could describe it's function a little more. Maybe `local_pass_file` or something

Reply:
Also, if this is false, will anything work?